### PR TITLE
Init Script Updates

### PR DESCRIPTION
**Pull request by jantaylor**

### CODE REVIEW

File: init.sh [hunk 1]
```diff
@@ -1,14 +1,123 @@
...
+echo "Checking for dotnet tool 'user-secrets'."
+if dotnet user-secrets --version; then
+    echo "'user-secrets' is installed."
+else
+    echo "Installing dotnet tool 'dotnet-user-secrets'..."
+    dotnet tool install --global dotnet-user-secrets
+fi
+
+## Set up JwtKey with defined key or random key TODO: Ignore it
+read -p "Input JwtKey or press (enter) for a random key:" key
```
Comment (snaveevans) [inline init.sh]: The key has to be longer than 16 characters, no need to check if it isn't but we should echo that to the user.

Comment (snaveevans) [reply init.sh]: I have to leave a comment to mark for `Requested Changes`

Comment (jantaylor) [reply init.sh]: I apparently didn't notified you had reviewed the PR so I'll go through that.

**Comment by jantaylor** [comment]
@snaveevans Updated the PR with the 16 character minimum in the echo. I'm not sure how to request another review without "dismissing" your current review?

**Pull request merged**